jetcd-launcher/src/main/java/io/etcd/jetcd/launcher/EtcdClusterFactory.java

```diff
@@ -21,10 +21,15 @@
 import edu.umd.cs.findbugs.annotations.NonNull;
 import java.net.URI;
 import java.util.List;
+import java.util.Set;
 import java.util.concurrent.CountDownLatch;
 import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.stream.Collectors;
 import java.util.stream.IntStream;
 import org.testcontainers.containers.Network;
+import org.testcontainers.shaded.com.google.common.base.Joiner;
+import org.testcontainers.shaded.io.netty.util.internal.ConcurrentSet;
```

Comment:
the newly add imports don't seem to be used anywhere.

Reply:
Done.